packages/ilmomasiina-backend/src/cron/deleteUnconfirmedSignups.ts

```diff
@@ -31,7 +31,7 @@ export default async function deleteUnconfirmedSignups() {
         include: [
           {
             model: Event,
-            attributes: ["id", "openQuotaSize"],
+            attributes: ["id", "openQuotaSize", "openQuotaPrice"],
```

Comment:
The open quota is not something one can sign up for, but rather an overflow shared between quotas. The code currently assumes people don't care if they're `in-quota` or `in-open` and can move people between those.

Therefore, the price shouldn't change when placed into the open quota, and I don't think this needs to exist